### PR TITLE
add page on csv export

### PR DESCRIPTION
**Pull request by jmcook1186**
adds user doc on csv export and adds info to cli page

### CODE REVIEW

File: docs/users/how-to-export-to-csv.md [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+initialize:
+  outputs:
+    - csv
+```
+
+You can also add `- yaml` if you want to export to both `yaml` and `csv` simultaneously.
+
+## CLI command
+
+Then, you must select the metric you want to export to CSV. The name of that metric must be added top the savepath provided to the `--output` command in the CLI, after a hashtag.
```
Comment (manushak) [inline docs/users/how-to-export-to-csv.md:23]: typo in "top"

File: docs/users/how-to-export-to-csv.md [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+sidebar_position: 5
+---
+
+# How to export to CSV
+
+IF supports exporting data to CSV files. This provides users with a data format that enables visualization and data anlysis using standard data analysis tools.
+
+## Manifest config
+
+To export your data to a CSV file, you have to provide a small pieve of config data to your manifest file:
```
[the remark this answers is not in the document]
Comment (jmcook1186) [reply docs/users/how-to-export-to-csv.md:11]:
```suggestion
To export your data to a CSV file, you have to provide a small piece of config data to your manifest file:
```

File: docs/users/how-to-export-to-csv.md [hunk 3]
```diff
@@ -0,0 +1,468 @@
+---
+sidebar_position: 5
+---
+
+# How to export to CSV
+
+IF supports exporting data to CSV files. This provides users with a data format that enables visualization and data anlysis using standard data analysis tools.
```
[the remark this answers is not in the document]
Comment (jmcook1186) [reply docs/users/how-to-export-to-csv.md:7]:
```suggestion
IF supports exporting data to CSV files. This provides users with a data format that enables visualization and data analysis using standard data analysis tools.
```

File: docs/users/how-to-export-to-csv.md (hunk 1 above)
Comment (jmcook1186) [reply docs/users/how-to-export-to-csv.md:23]:
```suggestion
Then, you must select the metric you want to export to CSV. The name of that metric must be added to the savepath provided to the `--output` command in the CLI, after a hashtag.
```

File: docs/users/how-to-export-to-csv.md [hunk 4]
```diff
@@ -0,0 +1,468 @@
...
+## Walkthrough
+
+You can start with a demo yaml, such as the following (save this as `pipeline-demo.yml`):
+
+```yaml
+name: pipeline-demo
+description:
+tags:
+initialize:
+  output:
```
Comment (narekhovhannisyan) [inline docs/users/how-to-export-to-csv.md:388]: @jmcook1186  outputs

Comment (jmcook1186) [reply docs/users/how-to-export-to-csv.md:388]:
```suggestion
  outputs:
```

**Review by narekhovhannisyan — APPROVED** [review]

**Pull request merged**